test/index.spec.ts

```diff
@@ -433,6 +437,64 @@ describe('Avro mock data generator', () => {
     });
   });
 
+  it('support namespace declared inside an array', () => {
+    const schema = {
+      name: 'Farms',
+      namespace: 'com.farms',
+      type: 'record',
+      fields: [
+        {
+          name: 'birds',
+          type: [
+            {
+              type: 'array',
+              items: {
+                name: 'bird',
+                type: 'record',
+                fields: [
+                  {
+                    name: 'size', type: [
+                      {
+                        namespace: 'com.farms',
```

Comment:
Nitpick: would you mind using a different namespace between here and line [443](https://github.com/ovotech/avro-mock-generator/pull/15/files#diff-23a10c932016d72b4787fa832989590f89d73fc6ef590e67d269fe7af962aaffR443)?

This will help make sure future iterations choose the correct namespace for building the full name.

Reply:
Thanks for your quick feedback, the PR is up-to-date 🤞